Cargo.toml

```diff
@@ -452,7 +451,7 @@ solana-loader-v4-interface = "=2.2.1"
 solana-loader-v4-program = { path = "programs/loader-v4", version = "=2.2.6" }
 solana-local-cluster = { path = "local-cluster", version = "=2.2.6" }
 solana-log-collector = { path = "log-collector", version = "=2.2.6" }
-solana-logger = "=2.2.1"
+solana-logger = "=2.3.1"
```

Comment:
`solana-transaction` has a [strict patch level dependency](https://crates.io/crates/solana-transaction/2.2.2/dependencies) on v2.3.1 so had to bump